unity_search_pages/src/Controller/SearchPagesController.php

```diff
@@ -76,6 +77,22 @@ public function getTitle($route = NULL) {
         return $title;
       }
     }
+    // Applying the term being viewed to Judiciary Sentencing guidelines page title.
+    elseif ($route === 'view.judicial_decisions_search.sentence_guide_search_page') {
+      $current_path = \Drupal::service('path.current')->getPath();
```

Comment:
We shouldn't call services directly from \Drupal in a controller, dependency injection should be used in this case.
You should be able to use the ContainerInterface to get the result that you need, see how it is done in the IFrame class (IFrame.php within the entity browser contrib module)